### PR TITLE
Add -u option to open browser with custom url

### PR DESCRIPTION
**Pull request by hugomn**
In some cases you have a aliased domain for your localhost (e.g: local.mydomain.com) it's necessary you open the browser after running the http-server pointing to that domain.

### CODE REVIEW

**Comment by indexzero** [comment]
Could you also update the README.md?

**Comment by hugomn** [comment]
Done! =)

**Comment by hugomn** [comment]
Any news on this?

**Comment by BigBlueHat** [comment]
This seems useful, but we may need to change the parameter from `-u` to `--url` after some of the other things like #245 get addressed--to be sure we're not going to make things too confusing parameter-wise. It's queued for v0.10.0 though.

File: README.md [hunk 1]
```diff
@@ -60,7 +60,9 @@ This will install `http-server` globally so that it may be run from the command
 
 `--cors` Enable CORS via the `Access-Control-Allow-Origin` header
 
-`-o` Open browser window after staring the server
+`-o` Open browser window after starting the server
+
+`-u` Url to display in the opened browser (in case you have an alias to your localhost)
```
Comment (thornjad) [inline README.md]: could we actually change this to `--url` to avoid confusion with `--utc` and `--username`?

**Comment by thornjad** [comment]
Actually @hugomn, can this functionality be achieved with the `-o` option?

File: README.md (hunk 1 above)
Comment (hugomn) [reply README.md]: No, it's not possible because `-o` doesn't let you specify the desired url to open. I'll check the conflicts (3 years after!).

**Comment by thornjad** [comment]
> No, it's not possible

What about using `-a`? Like with the URL in your first comment, could you do this with `-a local.mydomain.com -o /my/path`?

> (3 years after!)

Yeah I know, we're working on it :)

**Comment by thornjad** [comment]
Closing due to inactivity, re-open if we'd still like to get this functionality. In my opinion, this would be better added as part of `-o` rather than as its own option.

**Pull request closed without merging**